app/controllers/api/v1/listings_controller.rb

```diff
@@ -0,0 +1,43 @@
+module Api
+  module V1
+    class ListingsController < ApiController
+      before_action :authenticate_user!
```

Comment:
move this line to the ApiController. That way all endpoints are authenticated by default